### DISCUSSION TITLE
post/antiNotepadpp/

### DISCUSSION THREAD

**Discussion by giscus** [WhatDamon/whatdamon.github.io#3] [General]
# post/antiNotepadpp/

⚠️注意！本文可能会涉及政治相关内容，Notepad++现在那些反华页面和下载页面统统不能访问⚠️ 在开始吐槽Notepad++前，我们先介绍一下它的作者吧 作者简介侯今吾（英语：DonHO），台湾人，是纯文字编辑器Notepad++（一个GNU通用公共许可证的自由软件）的开发者。并且是自由软件的推动者之一。毕业于台湾淡江大学法文系及巴黎第七大学电脑工程硕士与电脑工程高等分科教育文凭。现在他的职业

https://damon233.js.org/post/antiNotepadpp/

**Comment by wanzm1105** [comment]
同意的

**Comment by ByYogi** [comment]
> 市面上的替代品可以让你眼花缭乱，为什么还要拘泥于这托屎呢？

有哪些替代品呢？我尝试找替代品，却发现无可替代。
最有潜力的是 `ndd(Notepad--)`，但是目前也只是有潜力罢了。

**Reply by caiming123** [reply]
https://github.com/dail8859/NotepadNext

**Reply by daoya** [reply]
Sublime text, Ultra Edit, Notepad3, Notepad4, Sakura Editor, Hidemaru Editor......这些windows下都可以用。
兄弟，AI时代了，找东西的能力要加强啊！
顺便说一下，在台独言论之前，我就不用Notepad++了，这东西能力不行，还不如小日本的Sakura Editor，你找个超大文本打开试试就知道了。

**Reply by ByYogi** [reply]
> https://github.com/dail8859/NotepadNext

这个可以，谢谢兄弟。我试着用这个平替

**Reply by ByYogi** [reply]
> Sublime text, Ultra Edit, Notepad3, Notepad4, Sakura Editor, Hidemaru Editor......这些windows下都可以用。 兄弟，AI时代了，找东西的能力要加强啊！ 顺便说一下，在台独言论之前，我就不用Notepad++了，这东西能力不行，还不如小日本的Sakura Editor，你找个超大文本打开试试就知道了。

你说的这些编译器我都知道，完全不能比好吧。
要不就是界面不好看，要不就是功能少不好用。还有就是性能差，打开一个文件慢。 Notepad++也就拿大文件说事了

**Comment by liudidi** [comment]
说一点，软件收费并不违反GPL
